### ISSUE TITLE
Default Icon Failing to Show

### ISSUE THREAD

**Issue by doug-ol** [jupyter-widgets/ipyleaflet#548]
I'm trying to use the [sample code ](https://ipyleaflet.readthedocs.io/en/latest/api_reference/marker.html)

```
from ipyleaflet import Map, Marker

center = (52.204793, 360.121558)

m = Map(center=center, zoom=15)

marker = Marker(location=center, draggable=False)
m.add_layer(marker);

m
```

This shows the map, but no icon shows.

![image](https://user-images.githubusercontent.com/33734195/79251084-8e5bc700-7e34-11ea-8c19-3a5e53383c38.png)

If I use a custom icon like this, it works:
```
from ipyleaflet import Marker, Icon, Map

center = (52.204793, 360.121558)

m = Map(center=center, zoom=10)
icon = Icon(icon_url='https://leafletjs.com/examples/custom-icons/leaf-green.png', icon_size=[38, 95], icon_anchor=[22,94])
mark = Marker(location=center, icon=icon, rotation_angle=90, rotation_origin='22px 94px')
m.add_layer(mark);

m
```

![image](https://user-images.githubusercontent.com/33734195/79251140-9e73a680-7e34-11ea-80d7-efe03b079477.png)

**Comment by martinRenou** [comment]
Do you see a 404 error in the JavaScript console? https://webmasters.stackexchange.com/questions/8525/how-do-i-open-the-javascript-console-in-different-browsers

**Comment by doug-ol** [comment]
Yes 
Yes this is what I see in the Chrome Debugger.
```
[object%20Module]:1 GET http://localhost:8889/[object%20Module] 404 (Not Found)
Image (async)
_createImg @ leaflet-src.js:7170
_createIcon @ leaflet-src.js:7137
createIcon @ leaflet-src.js:7118
_initIcon @ leaflet-src.js:7593
_initIcon @ leaflet.rotatedMarker.js:23
onAdd @ leaflet-src.js:7500
_layerAdd @ leaflet-src.js:6617
whenReady @ leaflet-src.js:4477
addLayer @ leaflet-src.js:6679
(anonymous) @ Map.js:140
Promise.then (async)
add_layer_model @ Map.js:139
e.update @ viewlist.js:54
(anonymous) @ Map.js:195
Promise.then (async)
render_leaflet @ Map.js:194
Promise.then (async)
render @ Map.js:190
(anonymous) @ manager-base.js:93
Promise.then (async)
(anonymous) @ manager-base.js:87
Promise.then (async)
e.create_view @ manager-base.js:86
e.display_model @ manager-base.js:73
renderModel @ renderer.js:56
async function (async)
renderModel @ renderer.js:30
createRenderedMimetype @ widget.js:429
createOutputItem @ widget.js:401
_insertOutput @ widget.js:384
onModelChanged @ widget.js:216
m @ index.js:478
c @ index.js:435
e.emit @ index.js:108
_onListChanged @ model.js:231
m @ index.js:478
c @ index.js:435
e.emit @ index.js:108
push @ observablelist.js:135
_add @ model.js:207
add @ model.js:128
_onIOPub @ widget.js:91
_handleIOPub @ future.js:216
async function (async)
_handleIOPub @ future.js:212
handleMsg @ future.js:186
_handleMessage @ default.js:1087
(anonymous) @ default.js:94
Promise.then (async)
_onWSMessage @ default.js:91
```

**Comment by martinRenou** [comment]
Which browser are you using?

**Comment by martinRenou** [comment]
What is your setup? I see you are using Chrome, I suspect your are using JupyterLab from the screenshot. Which ipyleaflet version is it?

**Comment by doug-ol** [comment]
JupyterLab Version 2.1.0  Chrome 80, Mac OSX Catalina 10.15.4

`ipyleaflet in /usr/local/lib/python3.7/site-packages (0.12.3)`

**Comment by martinRenou** [comment]
Should be fixed by #552
